weblate/trans/models/component.py

```diff
@@ -931,7 +931,8 @@ def install_autoaddon(self):
                 continue
 
             if (
-                addon.project_scope
+                hasattr(addon, "project")
+                and addon.project
```

Comment:
I think this code should have stayed as it was as it doesn't work with a model here.

Reply:
CI was failing for not having project_scope attribute for the class and throwing AttributeError. Added this change by removing current state for clarity and avoiding AttributeError.
```suggestion
            getattr(addon, "project_scope", False)        
```